cmd/podman/login.go

```diff
@@ -87,39 +93,25 @@ func loginCmd(c *cli.Context) error {
 	// username of user logged in to server (if one exists)
 	userFromAuthFile, passFromAuthFile, err := config.GetAuthentication(sc, server)
 	if err != nil {
-		return errors.Wrapf(err, "error getting logged-in user")
+		return errors.Wrapf(err, "error reading auth file")
```

Comment:
I know it's post merge, but any chance could we list where the authfile location is that we were trying to read?

Reply:
I think that's something `GetAuthentication` should do. In theory, there is a slice of files its trying but it doesn't continue in case of an error but returns it. Not sure that's correct.